### DISCUSSION TITLE
New camera controls and new shader.

### DISCUSSION THREAD

**Discussion by iulianiuga** [hjoykim/THREE#59] [Ideas]
Hi, 
Is there any chance to implement the MapControl (https://threejs.org/examples/?q=map#misc_controls_map) and maybe the EDL shader (https://www.cloudcompare.org/doc/wiki/index.php/EDL_(shader)) for point clouds as effect?
Thank you

**Comment by hjoykim** [comment]
I added MapControls and example.

**Comment by hjoykim** [comment]
hello
Can you tell me an example site to refer to EDL Shader?

**Reply by iulianiuga** [reply]
Here is an example, https://github.com/potree/potree/. The vizualisation made by potree using EDL can be seen Examples section.
Thank you for your effort, map control is perfect
